visualisation/3dSDPViewer.cpp

```diff
@@ -428,19 +428,16 @@ int main( int argc, char** argv )
     }
     if(vm.count("addMesh"))
     {
-      DGtal::Color meshColor = DGtal::Color::White;
-      if(vm.count("customColorMesh")){
+      bool customColorMesh =  vm.count("customColorMesh");
```

Comment:
It was just for the boolean in code, not for the option name.
This could be easier to understand for external peoples.